packages/discord-bot-ui/vite.config.ts

```diff
@@ -0,0 +1,35 @@
+/// <reference types='vitest' />
+import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin'
```

Comment:
I see a lot of changes in the discord bot. Is that actually working again?

Reply:
Most of the changes are around tooling, version updates and formatting. It needs some more work to be brought back up to date.